dataeng/jobs/analytics/SnowflakeMicrobachelorsITK.groovy

```diff
@@ -51,7 +51,10 @@ class SnowflakeMicrobachelorsITK {
                 }
             }
             triggers {
-                cron('H 6 * * 1-5')
+                buildResult('H 6 * * 1-5') {
+                    combinedJobs()
```

Comment:
Do I need this if there’s only one job?

Reply:
I don't think we need this if it's only 1 job